includes/feed/fetch/class-fetch.php

```diff
@@ -79,7 +70,7 @@ public function get_posts() {
 				'cata-blocks',
 				'Unable to fetch posts from remote.',
 				array(
-					'url' => $this->url,
+					'url' => $this->url
```

Comment:
There are multiple changes here just for removing commas, but I think the commas are in the code standard.